client/question/components/LearnerViewDirective.js

```diff
@@ -408,18 +355,16 @@ tie.directive('learnerView', [function() {
           border: 1px solid #333;
         }
         .tie-run-button {
+          background-color: rgb(42, 112, 232);
+          border: 1px solid rgb(32, 100, 200);
+          box-shadow: inset 0 1px 2px rgba(0,0,0.3);
           float: right;
           margin-top: 10px;
           position: relative;
         }
         .tie-run-button:hover {
           box-shadow: inset 0 1px 2px rgba(0,0,0.3);
         }
-        .tie-run-button:active {
-          background-color: rgb(42, 112, 232);
-          border: 1px solid rgb(32, 100, 200);
-          box-shadow: inset 0 1px 2px rgba(0,0,0.3);
-        }
```

Comment:
I think it's OK to have an :active rule, to distinguish it from hover over. Usually, a darker shade of the same color is used though (I think we kept the dark blue color from prior versions, though I'm not certain that was intentional--darker green for example would make sense for "I think I''m done".

Reply:
Whoops, sorry, I think I got a little aggressive with removing "dead" code but I put it back now! I actually made some changes to the button hovering (including what happens in dark mode), but kept the general idea of it getting darker for hover and then even a bit darker for active. The dark mode has only the border highlighted on hover (similar to the reset and theme buttons in dark mode) but also turns a bit darker on press. I am happy to change color/affect if anyone has objections.

Reply:
I split this part up into their respective color CSS (since we were using blue for both).